scripts/map_peaks_to_known_genes.py

```diff
@@ -16,9 +19,9 @@
 """
 epilog = ''
 parser = OptionParser(usage=usage,description=description,epilog=epilog)#,formatter=MultiLineHelpFormatter())
-parser.add_option('--upstream-window',dest='upst_win',type='int',default=100000,help='window width in base pairs to consider promoter region [default: %default]')
-parser.add_option('--downstream-window',dest='dnst_win',type='int',default=0,help='window width in base pairs to consider downstream region [default: %default]')
-parser.add_option('--tss',dest='tss',action='store_true',default=False, help='calculate downstream window from transcription start site instead of transcription end site')
+parser.add_option('--upstream-window',dest='upst_win',type='int',default= 2000,help='window width in base pairs to consider promoter region [default: %default]')
```

Comment:
I updated `HISTORY.rst` with the ongoing changes since the 0.3.0 release.  This pull request can now add to that list.